embedded-graphics/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 ### Changed
 
+- **(breaking)** [#256](https://github.com/jamwaffles/embedded-graphics/pull/256) Make `PrimitiveStyleBuilder` and `TextStyleBuilder` consuming when `.build()` is called.
```

Comment:
This isn't accurate anymore, because all methods now take ownership of the builder.
Will you also add links for the other issue numbers in the changelog?

Reply:
> This isn't accurate anymore, because all methods now take ownership of the builder.

True, fixed.

> Will you also add links for the other issue numbers in the changelog?

Yeah, before release probably. Should be a simple find/replace.